server/utils/helperFunctions.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Contents
+ * Public:
+ *  - checkQueryForEmptyFields()
+ *  - updateRequest()
+ *  - validateQuery()
+ * Private:
+ *  - updateRequestField()
+ *  - isObjectIdValid()
+ *  - isArrayBoolean()
+ */
```

Comment:
Nice - but I don't think we need this here since you're already defining documentation in the helpers.